### ISSUE TITLE
Emotes and more please

### ISSUE THREAD

**Issue by AllRanger** [ArtimosPenguidor/animation-transmog#2]
I love the plugin and was wondering if the other in-game emotes like"wave, yes, no, dance, jig, think, salute"..ect could be added to where the player repeats them in the miscellaneous movement mode?

**Comment by ArtimosPenguidor** [comment]
Are you meaning like setting an animation (like wave) as an idle animation (like the player periodically waves while just standing still)? Or are you meaning over riding the walking animation with a dance or other emote?

**Comment by AllRanger** [comment]
The animation just repeats its self over and over while standing still

**Comment by ArtimosPenguidor** [comment]
Yeah that sounds like a great idea, though I'd probably make it play at random time intervals (like 10-20 seconds) while idle so that it seems a bit more organic. 

I kinda dropped the ball on updating this plugin because I was running out of inspiration, but your request seems like the perfect excuse to get back into updating the plugin. 

I'll update this issue once that functionality is added in!

**Comment by AllRanger** [comment]
Hey that's great SO happy to re-inspire you! I could see it being used for a lot more!

 I was trying to make a short clip of me just waving without the UI active and this plug-in would be of help to me and alot! And of course other content creators Indeed!

**Comment by ArtimosPenguidor** [comment]
If that's the case, I can also add a feature that just let you select an emote and loop it constantly, for content creation purposes.

**Comment by AllRanger** [comment]
Yes! That would be really great!
